_data/simple-icons.json

```
@@ -16035,6 +16035,11 @@
             "hex": "7D4698",
             "source": "https://styleguide.torproject.org/brand-assets"
         },
+        {
+            "title": "Torizon",
+            "hex": "FAAF00",
+            "source": "https://docs.toradex.com/105996-torizon-logo-white.svg"
```

Use just the page where the image is found, please:

```suggestion
            "source": "https://toradex.com/torizon"
```